package.json

```
@@ -66,7 +66,8 @@
     "import:goals": "node ./build/server/tools/importTTAPlanGoals.js",
     "import:hses:local": "./node_modules/.bin/babel-node ./src/tools/importGrantGranteesCLI.js --skipdownload",
     "import:hses": "node ./build/server/tools/importGrantGranteesCLI.js",
-    "reconcile:legacy": "./node_modules/.bin/babel-node ./src/tools/reconcileLegacyReports.js"
+    "reconcile:legacy": "./node_modules/.bin/babel-node ./build/server/tools/reconcileLegacyReports.js",
```

This doesn't need babel-node since it's already gone through translation.

```suggestion
    "reconcile:legacy": "node ./build/server/tools/reconcileLegacyReports.js",
```